test/scenarios/host_group_vars/molecule/default/molecule.yml

```diff
@@ -16,6 +16,9 @@ platforms:
       - example_1
 provisioner:
   name: ansible
+  config_options:
```

Comment:
Note to ourselves: let's make a `molecule.yml` in `test/functional/scenarios` which holds this default configuration and change `molecule` invocations to include `--base-config ...`.